src/containers/ImportResources/ImportResources.scss

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.bx--text-input__field-wrapper {
+  width: 100%;
+}
+
+.bx--label {
+  font-size: 1rem !important;
+  margin-top: 0.8rem;
+}
+
+.bx--form-requirement {
+  margin-bottom: 0 !important
+}
+
+.saDropdown {
+  margin: 0% !important;
+  width: 100% !important;
+  border-bottom: 1px solid #8c8c8c !important;
```

Comment:
use design tokens where possible instead of hard-coding colours etc.
https://www.carbondesignsystem.com/guidelines/color/usage

Reply:
I had tried originally to use the design tokens but they didn't seem to be recognised, how do I use them/is there something else I should do for them to be recognised?

Reply:
`@import '~carbon-components/scss/globals/scss/vars';` at the top of the file you want to use them in